<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LogIn</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>530</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>65</y>
     <width>300</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="email_input">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border:1px solid rgb(240,243,246);
	border-radius:10px;
	font: 13pt &quot;Andale Mono&quot;;
	color:rgb(43,59,78);
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_input">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border:1px solid rgb(240,243,246);
	border-radius:10px;
	font: 13pt &quot;Andale Mono&quot;;
	color:rgb(43,59,78);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Andale Mono&quot;;
color:rgb(43,59,78);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 13pt &quot;Andale Mono&quot;;
color:rgb(43,59,78);</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="login_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 146, 212);
	color:rgb(243,247,254);
	border: 0px solid;
	border-radius: 10px;
	font: 10pt &quot;Andale Mono&quot;;
}

QPushButton:hover {
	background-color: rgb(243,247,254);
	color:rgb(59, 146, 212);
}</string>
    </property>
    <property name="text">
     <string>LOG IN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>237</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Andale Mono&quot;;
color:rgb(43,59,78);</string>
    </property>
    <property name="text">
     <string>Don't have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_btn">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>230</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color:rgb(59, 146, 212);;
	border: 0px solid;
	border-radius: 10px;
	font: 10pt &quot;Andale Mono&quot;;
}

</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;Andale Mono&quot;;
color:rgb(43,59,78);</string>
    </property>
    <property name="text">
     <string>SLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
